From: Chun-wei Fan <fanchunwei@src.gnome.org>
Date: Wed, 18 May 2022 15:29:24 +0800
Subject: gtk/Makefile.am: Fix distcheck

Apparently distcheck expects the generated gtk.gresource.xml to be
there.

Origin: upstream, 3.24.35, commit:bac9ff22f57891f71ebd51bb650d2c783421131a
---
 gtk/Makefile.am | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/gtk/Makefile.am b/gtk/Makefile.am
index 8775ced..fc701d1 100644
--- a/gtk/Makefile.am
+++ b/gtk/Makefile.am
@@ -964,6 +964,9 @@ endif
 
 EXTRA_DIST += $(gtk_all_private_h_sources) $(gtk_extra_sources)
 
+# We need to dist this for `make distcheck`, apparently...
+EXTRA_DIST += gtk.gresource.xml
+
 
 pkgdatadir = $(datadir)/gtk-$(GTK_API_VERSION)
 pkgdata_DATA = gtkbuilder.rng
